### DISCUSSION TITLE
LCD pins

### DISCUSSION THREAD

**Discussion by yaleman** [TeamOpenFIRE/OpenFIRE-Firmware#14] [Q&A] [answered]
The readme mentions SSD1306 LCD's but none of the board pinouts show where to connect it - do they connect to the same SCL/SDA pins as the camera?

**Comment by SeongGino** [comment, marked as answer]
Keep in mind the disclaimer on that file:

> #### NOTE: Any button/function GPIO pin assignments written here are the defaults, and can be changed for any other function from the OpenFIRE App.

With that said, OLED is connected to *Peripheral SCL/SDA*, which is not part of any standard layout.

**Reply by yaleman** [reply]
Ah, didn't pick up on the implications of the "peripherals" designation. Thanks!

**Reply by SeongGino** [reply]
That's because non-camera related accessories using I2C will be sharing that second "peripherals" line, e.g. a gyroscope module will, in theory, share space with the OLED. It's a future proofing method, and in future if/when additional secondary peripherals are added, then their active state will be determined by toggles instead.
